components/merkledb/examples/manual_migration.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Shows how to provide database data migration manually.
+//! The main logic of this example described in the `manual_migration` and `migrate_wallets` functions.
```

Comment:
Nits:

- Overly long line here and in some other doc comments. Also, you should split doc comments into paragraphs (using empty lines) for readability.
- "The main logic of this example **is** described..."